kumascript/src/errors.js

```diff
@@ -178,6 +180,20 @@ class MacroBrokenLinkError extends MacroExecutionError {
   }
 }
 
+/**
+ * A MacroDeprecatedError is a special case of MacroExecutionError.
+ */
+class MacroDeprecatedError extends MacroExecutionError {
```

Comment:
Seems to work to search for
<img width="1227" alt="Screen Shot 2020-06-25 at 8 04 13 PM" src="https://user-images.githubusercontent.com/26739/85807176-186eb880-b71f-11ea-881e-2cad159027f7.png">
Great!